src/sonic-host-services/scripts/hostcfgd

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
```

Comment:
Why do you want to change this?

Reply:
weird, I don't remember this change "#!/usr/bin/python3". I will check it.